README.md

```diff
@@ -1 +1,181 @@
+- Install the `npm` package manager use this [to install both node and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+- [A terminal](https://code.visualstudio.com/docs/terminal/basics).
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone https://github.com/shihabrafio/Space-Travelers-Hub.git
+```
+
+### Install
+
+To install all dependencies, run:
+
+```
+  npm install
+```
+
+### Usage
+
+After Cloning this repo to your local machine.
+To get it running on your default browser and local host, run:
+
+```
+  npm start
+```
+
+### Run tests
+
+To run tests, run:
+
+```
+  npm test
+```
+
+### Deployment
+
+You can deploy this project using, Netlify:
+
+- I used Netlify to delpoy the project.
+- For more information, check [Netlify](https://www.netlify.com/) website.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👤 Authors <a name="authors"></a>
+
+👤 **A.I.M. Shihab Uddin Khan**
+
+- GitHub: [@shihabrafio](https://github.com/shihabrafio)
+- Twitter: [@shihab_uddin19](https://twitter.com/shihab_uddin19)
+- LinkedIn: [Shihab Uddin Khan](https://www.linkedin.com/in/shihab-uddin-khan-45620a16a/)
+
+👤 **Vangelis Fotiadis**
+
+- GitHub: [@vangelif](https://github.com/vangelif)
+- Twitter: [@vangfot](https://twitter.com/vangfot)
+- LinkedIn: [Vangelis Fotiadis](https://www.linkedin.com/in/vangfot/)
+
+> 👤 **Yousef Badr**
+
+- GitHub: [@ybadr99](https://github.com/ybadr99)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/yousef-mohamed-badr/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Enable Search Feature**
+- [ ] **Imporove responsitivity**
+- [ ] **Style Myprofile page better**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a Start ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Thanks to the Microverse team for the great curriculum.
+- Thanks to the Code Reviewer(s) for the insightful feedbacks.
+- A great thanks to My coding partner(s), morning session team, and standup team for their contributions.
+- Hat tip to anyone whose code was used.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Can I Fork it and Imporove on it**
+
+  - Yes, You can, feel free.
+
+- **Does the API respond always**
+
+  - Yes, always.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./License) licensed.
```

Comment:
- [ ] Kindly fix the MIT license link; now, when I press it, I get a 404 not found error.